build.gradle

```diff
@@ -33,7 +33,7 @@ buildscript {
 //
 // It must be false for fat aar generation of MuxExoPlayer to work
 // This is a bug/problem with the fat aar mechanism
-def inLocalDevModeFlag = false
+def inLocalDevModeFlag = true
```

Comment:
```suggestion
def inLocalDevModeFlag = false
```
... right?

Reply:
I have to admit I'm slightly confused by the ongoing presence of inLocalDevModeFlag - it's not in the master branch.

Has something gone wrong with the merging etc. here or it's being reintroduced deliberately?

Reply:
I imagine this was included by accident, setting it to `false` seems right

Reply:
I can clean it up in the next version, along with applying the new approach for the module dependencies.